source/component/PasDoc_Versions.pas

```diff
@@ -140,28 +140,24 @@ implementation
 function COMPILER_NAME: string;
 begin
   COMPILER_NAME :=
+    { FreePascal }
     {$IFDEF FPC}
     'FPC ' + Format('%d.%d.%d', [FPC_VERSION, FPC_RELEASE, FPC_PATCH]);
     {$define COMPILER_VERSION_DEFINED}
     {$ENDIF}
-
-    {$IFDEF KYLIX_1}   'KYLIX 1';      {$define COMPILER_VERSION_DEFINED} {$ENDIF}
-    {$IFDEF KYLIX_2}   'KYLIX 2';      {$define COMPILER_VERSION_DEFINED} {$ENDIF}
-    {$IFDEF KYLIX_3}   'KYLIX 3';      {$define COMPILER_VERSION_DEFINED} {$ENDIF}
-    {$IFDEF DELPHI_15} 'DELPHI XE';    {$define COMPILER_VERSION_DEFINED} {$ENDIF}
-    {$IFDEF DELPHI_14} 'DELPHI 2010';  {$define COMPILER_VERSION_DEFINED} {$ENDIF}
-    {$IFDEF DELPHI_12} 'DELPHI 2009';  {$define COMPILER_VERSION_DEFINED} {$ENDIF}
-    {$IFDEF DELPHI_11} 'DELPHI 2007';  {$define COMPILER_VERSION_DEFINED} {$ENDIF}
-    {$IFDEF DELPHI_10} 'DELPHI 2006';  {$define COMPILER_VERSION_DEFINED} {$ENDIF}
-    {$IFDEF DELPHI_9}  'DELPHI 2005';  {$define COMPILER_VERSION_DEFINED} {$ENDIF}
-    {$IFDEF DELPHI_7}  'DELPHI 7';     {$define COMPILER_VERSION_DEFINED} {$ENDIF}
-    {$IFDEF DELPHI_6}  'DELPHI 6';     {$define COMPILER_VERSION_DEFINED} {$ENDIF}
-    {$IFDEF DELPHI_5}  'DELPHI 5';     {$define COMPILER_VERSION_DEFINED} {$ENDIF}
-    {$IFDEF DELPHI_4}  'DELPHI 4';     {$define COMPILER_VERSION_DEFINED} {$ENDIF}
-
-    {$IFNDEF COMPILER_VERSION_DEFINED}
-    { As a fallback, use Delphi CompilerVersion. }
+    { Delphi 6+ }
+    {$IF CompilerVersion > 14.0}
     Format('DELPHI %2.1f', [CompilerVersion]);
```

Comment:
Please change above line to

```
Format('DELPHI (Compiler Version %2.1f)', [CompilerVersion]);
```

The reason is that `CompilerVersion` may not be obvious to Delphi users. The `CompilerVersion` is not what is usually used when speaking about Delphi versions. 
    
E.g. Delphi 11.0 = product version 28 = compiler version 35. See https://docwiki.embarcadero.com/RADStudio/Sydney/en/Compiler_Versions , https://delphi.fandom.com/wiki/CompilerVersion_Constant . So people using Delphi 11.0 will not see `11.0`, they will see `35.0`.

( I know that you *did not* change the line above, but I would ask to change it in the course of this PR -- because your changes mean that this line is now more significant, it will be used most of the time. )
  
I use it similarly in Castle Game Engine, https://github.com/castle-engine/castle-engine/blob/master/src/base/castleutils_miscella.inc#L271 .